src/ssl.c

```diff
@@ -12719,11 +12749,14 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
                     neededState = SERVER_HELLOVERIFYREQUEST_COMPLETE;
             #endif
             /* get response */
+            WOLFSSL_MSG("ssl.c ssl->options.serverState < neededState");
```

Comment:
Don't put code into messages.
Describe what is going on.
Better would be:
"Server state up to needed state"

Reply:
Agreed. Done.